#153756399 setup frontend testing

Victoradukwu opened the pull request:
#### What does this PR do?
>Setup frontend testing
#### Description of Task to be completed?
>Install Karma and Jasmine
>define karma.config file
>set a sample test file
#### How should this be manually tested?
>run **karma start** command from the terminal
#### Any background context you want to provide?
>N/A
#### What are the relevant pivotal tracker stories?
>153756399 Setup unit testing for the front end
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/24465059/34207650-ab06ae0e-e58b-11e7-88dd-f701410af797.png)
#### Questions:
>N/A

Victoradukwu committed:
chore(testing):[153756399] setup frontend testing using Karma with Jasmine

temilaj commented:
Good work raising this PR. 

One thing you can work on is your commit message. At the moment, it does not say the individual tasks you did to complete this chore?

temilaj commented on karma.conf.js:
```diff
@@ -0,0 +1,68 @@
+    // Concurrency level
+    // how many browser should be started simultaneous
+    concurrency: Infinity
+  })
```
Can you fix this?

Victoradukwu committed:
chore(testing): Edit karma.config file
>
> add semicolon to lines 67 and 68
> remove Firefox from the browser list
>
> Finishes [153756399]

Victoradukwu committed:
add firefox to list of browsers
